### PR TITLE
Shortening some examples to the minimum demonstrative length

### PR DESCRIPTION
**Pull request by venantius**
I've tried to tighten up a number of our examples to the shortest length that
reasonably captured the rule they were demonstrating.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,10 +68,6 @@ Airbnb also maintains a [JavaScript Style Guide][airbnb-javascript].
     # good
     def self.create_translation(phrase_id,
                                 phrase_key,
-                                target_locale,
-                                value,
-                                user_id,
-                                do_xss_check,
```
Comment (BMorearty) [inline README.md]: This one looks funky because after removing these params, they would all fit on one line.

File: README.md [hunk 2]
```diff
@@ -88,18 +76,14 @@ Airbnb also maintains a [JavaScript Style Guide][airbnb-javascript].
...
-      user_id,
-      do_xss_check,
       allow_verification
     )
       ...
     end
 
     # bad
-    def self.create_translation(phrase_id, phrase_key, target_locale,
-                                value, user_id, do_xss_check, allow_verification)
```
Comment (BMorearty) [inline README.md]: same here

File: README.md [hunk 3]
```diff
@@ -88,18 +76,14 @@ Airbnb also maintains a [JavaScript Style Guide][airbnb-javascript].
     def self.create_translation(
       phrase_id,
       phrase_key,
-      target_locale,
-      value,
-      user_id,
-      do_xss_check,
```
Comment (BMorearty) [inline README.md]: same here

**Comment by BMorearty** [comment]
LGTM except for the funky ones.

**Comment by venantius** [comment]
@BMorearty do you think there's a length that's shorter than the original for the funky example that is still demonstrative? I know it could all fit on one line at the moment but I tried to shorten it to the minimal possible example that demonstrates the underlying principle. Just wondering how much I should really be re-adding to get the point across without taking up a screen full of space.

**Comment by venantius** [comment]
We could also probably trim a few lines in that example by not forcing the rule to only apply to method definitions, since the same rules presumably apply to method calls as well.

**Comment by BMorearty** [comment]
If joining all the lines would be >100 chars, then you can split the lines and show the result in the sample. At least that's what I would do.

**Comment by tommydangerous** [comment]
Looks nice

**Comment by venantius** [comment]
I'm going to close this since it's not super important.

**Pull request closed without merging**